tests/test_date.py

```diff
@@ -282,12 +423,12 @@ def test_get_date_data_should_not_strip_timezone_info(self):
         self.assertTrue(hasattr(date_data['date_obj'], 'tzinfo'))
 
     def test_should_parse_date_with_timezones_using_format(self):
-        date_string = "2014/11/17 14:56 EDT"
-        date_format = "%Y/%m/%d %H:%M EDT"
-        expected = datetime(2014, 11, 17, 14, 56)
-        date_data = self.parser.get_date_data(date_string, date_formats=[date_format])
-        self.assertEqual('day', date_data['period'])
-        self.assertEqual(expected, date_data['date_obj'])
+        self.given_date_string("2014/11/17 14:56 EDT")
+        self.given_date_format("%Y/%m/%d %H:%M EDT")
+        self.given_expected_date(datetime(2014, 11, 17, 14, 56))
```

Comment:
What if instead this method we use `then_parsed_datetime_is(datetime(2014, 11, 17, 14, 56))` later in this test?

`then_parsed_datetime_is` is renamed `then_should_parse_date_with_expected_date`